tests/testthat/_snaps/derive_vars_dt.md

```diff
@@ -0,0 +1,61 @@
+# impute_dtc_dt Test 8: min_dates length mismatch provides error
+
+    Code
+      impute_dtc_dt(c("2020-12", NA_character_), min_dates = list(c(ymd("2020-12-06")),
+      c(ymd("2020-11-11"))), highest_imputation = "Y")
+    Condition
+      Error in `restrict_imputed_dtc_dt()`:
+      ! Length of `min_dates` do not match length of dates to be imputed.
```

Comment:
Is this error going to make sense to the user if it pops out in derive_vars_dt?  

I didn't have time to look deeper, but we don't export this function and so if a user saw this function as the culprit would that be confusing?

Reply:
Okay - maybe 1.2?  @ddsjoberg WDYT?